src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerAddAudioMetadataIntTest.java

```diff
@@ -146,7 +146,7 @@ void addAudioMetadataNonExistingCourthouse() throws Exception {
 
         String actualJson = mvcResult.getResponse().getContentAsString();
         String expectedJson = """
-            {"type":"COMMON_100","title":"Provided courthouse does not exist","status":400,"detail":"Courthouse 'TEST' not found."}""";
+            {"type":"100","title":"Provided courthouse does not exist","status":400,"detail":"Courthouse 'TEST' not found."}""";
```

Comment:
Just a suggestion but maybe less fragile to use the enumeration here, just incase the code changes in future. Same pattern would be needed for all integration tests.

Reply:
Yeah. Agree it could be improved. Ideally we should create the expectedJson with a builder and serialize to json. Perhaps we could do this on another housekeeping ticket as we would have to do it consistently across the board?